lib/demon/email_sync.rb

```diff
@@ -146,12 +149,6 @@ def kill_threads
 
   def after_fork
     log("[EmailSync] Loading EmailSync in process id #{Process.pid}")
-
-    loop do
```

Comment:
I think only an `EmailSync` process should run for a site, unlike Sidekiq which should run for each host of a site. I think this prevented that.